macros/GroupData.json

```diff
@@ -950,6 +969,23 @@
             "properties": [],
             "events":     [ "message" ]
         },
+        "Intersection Observer API": {
+            "interfaces": [ "IntersectionObserver",
+                            "IntersectionObserverEntry" ],
+            "methods":    [ "IntersectionObserver.disconnect",
```

Comment:
Should these have parenthesis, like ``IntersectionObserver.disconnect()``? It seems the majority of methods do.